packages/amplify-prompts/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "amplify-prompts",
+  "version": "1.0.0",
+  "description": "Utility functions for Amplify CLI terminal I/O",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "tsc",
+    "clean": "rimraf lib tsconfig.tsbuildinfo",
+    "demo": "yarn build && node lib/demo/demo.js",
+    "test": "jest",
+    "watch": "tsc -w"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/aws-amplify/amplify-cli.git"
+  },
+  "keywords": [
+    "amplify",
+    "cli",
+    "prompts"
+  ],
+  "author": "Amazon Web Services",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/aws-amplify/amplify-cli/issues"
+  },
+  "homepage": "https://github.com/aws-amplify/amplify-cli#readme",
+  "dependencies": {
+    "chalk": "^4.1.1",
```

Comment:
is it possible to update chalk in the rest of the packages to match 4.1.1? Or is it something we need to do over time once we refactor enough code to use this package over context.print.x?

Reply:
resolving this as it looks like https://github.com/aws-amplify/amplify-cli/pull/7823 will take care of it